components/esp_lvgl_port/esp_lvgl_port.c

```diff
@@ -158,6 +158,26 @@ esp_err_t lvgl_port_init(const lvgl_port_cfg_t *cfg)
     return ret;
 }
 
+esp_err_t lvgl_port_resume(void)
+{
+    if (lvgl_port_ctx.tick_timer != NULL) {
+        lv_timer_enable(true);
+        return esp_timer_start_periodic(lvgl_port_ctx.tick_timer, lvgl_port_timer_period_ms * 1000);
+    } else {
+        return ESP_ERR_INVALID_ARG;
+    }
```

Comment:
I am sorry, but it is not nice. Better can be second `return` without `else` OR put into `err = esp_timer_star_...` and then `return err` on the end of function;